content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -207,31 +207,35 @@ Once Pod is assigned to a node by scheduler, kubelet starts creating containers
     ...
    ``` 
 
-## Pod readiness gate
+## Pod readiness {#pod-readiness-gate}
 
 {{< feature-state for_k8s_version="v1.14" state="stable" >}}
 
-In order to add extensibility to Pod readiness by enabling the injection of
-extra feedback or signals into `PodStatus`, Kubernetes 1.11 introduced a
-feature named [Pod ready++](https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/0007-pod-ready%2B%2B.md).
-You can use the new field `ReadinessGate` in the `PodSpec` to specify additional
-conditions to be evaluated for Pod readiness. If Kubernetes cannot find such a
-condition in the `status.conditions` field of a Pod, the status of the condition 
-is default to "`False`". Below is an example:
+Your application can inject extra feedback or signals into PodStatus:
+_Pod readiness_. To use this, set `readinessGates` in the PodSpec to specify
```

Comment:
@sftim, I think it would be helpful to identify the feature gate name (possibly in addition to the concept Pod readiness). I see that @mbrow refers to the feature as `[Pod] readiness gates`. Perhaps phrase does make sense for the section title. I am interpreting `gates` as checks for readiness.

Possible additions/changes:
The`PodReadinessGates` feature extends the conditions that determine _Pod readiness_. Your application can inject extra feedback or signals into the `PodStatus`. You can set `readinessGates` in the `PodSpec` to create extra readiness checks for your Pod. The `readinessGates` field defines a list of additional conditions that the kubelet ...

Reply:
PodReadinessGates is GA since Kubernetes v1.14, so there is no feature flag named `PodReadinessGates` in the current release.
If I were adding details about how to enable it, I'd put that in a backport.

Reply:
OK. I thought it is worthwhile mentioning because the information (about the feature gate) is listed in the feature gates table.

Reply:
All feature gates are listed in that table, even for graduated features.